packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -220,7 +230,7 @@ export namespace CommonCommands {
     export const TOGGLE_STATUS_BAR: Command = {
         id: 'workbench.action.toggleStatusbarVisibility',
         category: VIEW_CATEGORY,
-        label: 'Toggle Status Bar Visibility'
+        label: 'Toggle Status Bar'
```

Comment:
The change should be reverted, the command in the command palette (<kbd>F1</kbd>) is `Toggle Status Bar Visibility`.
When registering the menu item for this command you would use `Toggle Status Bar`.